### ISSUE TITLE
Feature request: Add url to the authorizer

### ISSUE THREAD

**Issue by ogenodisho** [jerelmiller/redux-simple-auth#68]
I think you should be able to reason on the URL when deciding whether or not to add headers to the request.

A common example use case would be having different authorization tokens for different servers.

**Comment by colmbrady** [comment]
@ogenodisho see https://github.com/jerelmiller/redux-simple-auth/pull/71

Suggest you close this.

**Comment by ogenodisho** [comment]
@jerelmiller Yes. My use case is reasoning on the request URL when attaching auth headers to it. However, the `authorize` function doesn't pass the URL so I can't do that.
